### ISSUE TITLE
Fill in sendImageToDisplay()

### ISSUE THREAD

**Issue by wecodewithmichaelmccloskey** [wecodewithmichaelmccloskey/Smart_Breadboard#29]
- [ ] Fill in sendImageToDisplay()
- [ ] Edit demo file to be able to test function
- [ ] Verify that function is able to work

(1 hour)

**Comment by wecodewithmichaelmccloskey** [comment]
@Fatiha897 please review

**Comment by wecodewithmichaelmccloskey** [comment]
sendImageToDisplay() has been changed to addPicture(). This was done to better reflect what the function is actually doing now. The function was changed to simply add a picture to an existing image.

Side Note: I know the wording is a little confusing. That is because the library that I am using refers to everything that is sent to the display as an image. So I changed what I had been calling images to pictures. From now on, any time that I say (or code) picture, it means an actual picture, and if I say (or code) image, it refers to the object that stores all the information that will be sent to the display.

The changes to the function were made so that it was possible to add pictures and drawing (which includes text) to the same image, as this is likely the format that will be used when we are showing instructions.

Below is a test to verify the functionality of the function:
```
pi@raspberrypi23:~/Smart_Breadboard $ python3
Python 3.9.2 (default, Mar 12 2021, 04:06:34)
[GCC 10.2.1 20210110] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import Smart_Breadboard as sbb
>>> disp = sbb.setupDisplay()
>>> image = sbb.createNewImage(disp)
>>> sbb.addPicture(image, "Resistor.jpg", 0, 0, 320)
>>> sbb.sendToDisplay(disp, image)
>>> exit()
```

Output:


![Image](https://user-images.githubusercontent.com/114356709/222605751-8fa04448-0f5e-474d-a374-0ef819dad8b0.jpg)
